src/systems/filecoin_blockchain/storage_power_consensus/_index.md

```diff
@@ -112,7 +112,21 @@ Each Ticket should be generated from the prior one in the ticket-chain and verif
 {{< readfile file="storage_power_consensus_subsystem.id" code="true" lang="go" >}}
 {{< readfile file="storage_power_consensus_subsystem.go" code="true" lang="go" >}}
 
-### Repeated Leader Election attempts
+## Repeated Leader Election attempts
 
 In the case that no miner is eligible to produce a block in a given round of EC, the storage power consensus subsystem will be called by the block producer to attempt another leader election by incrementing the nonce appended to the ticket drawn from the past in order to attempt to find a new winning `PartialTicket` and trying again. 
-Note that a miner may attempt to grind through tickets by incrementing the nonce repeatedly until they find a winning ticket. However, any block so generated in the future will be rejected by other miners (with synchronized clocks) until that epoch's appropriate time.
\ No newline at end of file
+Note that a miner may attempt to grind through tickets by incrementing the nonce repeatedly until they find a winning ticket. However, any block so generated in the future will be rejected by other miners (with synchronized clocks) until that epoch's appropriate time.
+
+{{<label min_miner_size>}}
+## Minimum Miner Size
+
+{{% notice placeholder %}}
+The below values are currently placeholders.
+{{% /notice %}}
+
+In order to secure Storage Power Consensus, the system defines a minimum miner size required to participate in consensus.
+
+Specifically, miners must have either at least 100TB of active power (i.e. storage power currently used in storage deals) or 1/3 of the network's active storage power to participate in SPC.
+Miners smaller than this cannot mine blocks and earn block rewards. Put another way, this means that network liveness requires at least 3 active miners (running storage deals) or 300TB of active power.
```

Comment:
Why at least 3 active miners?  One miner with all power has > 1/3 power right?

Reply:
duh. went through a whole cycle of weird useless calculations clearly. Lesson not to spec late